tools/backupnode/backupnode.cpp

```diff
@@ -62,7 +62,7 @@ static bool inexactDateMatch = false;
 
 #ifdef USE_JLOG // and why not?
 #define println PROGLOG
-#define printerr ERRLOG
+#define printerr OERRLOG
```

Comment:
Why this change? I think most (all?) relate to use specified options, so not relevant to notify the operator